### ISSUE TITLE
Launch Player doesn't (launch a player)

### ISSUE THREAD

**Issue by brianjmurrell** [collinss/Cinnamon-Soundbox#18]
When I click Launch Player, nothing happens.  I have a Preferred Application for Music set to rhythmbox.

Is there anything else I need to do?

**Comment by collinss** [comment]
@brianjmurrell What version of Cinnamon are you using? Are you using the version from spices? If so, it is broken for Cinnamon 2.4 and later, and is very much out of date. I have made a lot of changes since then. If you want, you can try out the latest version here, though I plan on uploading a new version to spices in the next few days.

**Comment by brianjmurrell** [comment]
@collinss Cinnamon 2.4.  I am using the version of Soundbox from here on github.

**Comment by collinss** [comment]
Oh, I see. I think https://github.com/collinss/Cinnamon-Soundbox/commit/62f566134ce97c3e62acb0e7a17b3604f2963028 is causing the problem. I had to make this change to support the latest versions of Cinnamon on Github (which I use). I'll make sure to patch it to support 2.4 before I post the updated version to Spices. In the mean time, you could try making the relevant changes in reverse.

**Comment by collinss** [comment]
I just fixed it to work (I think) in Cinnamon 2.4, but I don't currently have that version on my computer. Could you please test it and make sure it actually works for you?

**Comment by brianjmurrell** [comment]
@collinss Yeah, I get a list of installed players now when I press the Launch Player button.  Great work!

**Comment by collinss** [comment]
Good to hear! Let me know if you find any problems. If not, I will probably release as-is by the end of the week.
